src/components/links/internal-link.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+import { Link } from 'gatsby';
+
+export const InternalLink = styled(Link)`
```

Comment:
I would put both links into the same `link.tsx` file. Makes it easier to manage them, as both have a bunch of stuff in common.

Reply:
+1